modules/comment/comment.service.js

```diff
@@ -1,28 +1,28 @@
-const Comments = require('./comments.model');
+const Comments = require('./comment.model');
 
 class CommentsService {
   getAllComments() {
     return Comments.find();
   }
 
-  getCommentsById(id) {
+  getCommentById(id) {
     return Comments.findById(id);
   }
 
   getAllCommentsByProduct(id) {
     return Comments.find({ product: id });
   }
 
-  updateCommentsById(id, comments) {
+  updateComment(id, comments) {
     return Comments.findByIdAndUpdate(id, comments);
```

Comment:
Please, add `{ new: true }` to return newly updated comment

Reply:
done